server/routerlicious/packages/services-client/src/restWrapper.ts

```diff
@@ -83,6 +83,14 @@ export class RestWrapper {
                     debug(`request to ${options.url} failed ${error ? error.message : ""}`);
                 }
 
+                if (error.response && error.response.status
+                    && error.response.data && error.response.data.retryAfter
+                    && error.response.status === 429 && error.response.data.retryAfter > 0) {
+                    setTimeout(async () => {
+                        return this.request<T>(options, statusCode);
+                    }, error.response.data.retryAfter * 1000);
```

Comment:
isn't the property name retryAfterInMs?
I could be wrong, but please verify.

Reply:
The `ThrottlingError` thrown by `Throttler` extends `INackContent` which has the property `retryAfter` (which is in seconds) to make it easily compatible with other existing uses of Throttling retry logic in client code

Reply:
For the ThrottlingError, retryAfter (second) is used: https://github.com/microsoft/FluidFramework/blob/6514933a426010f0ba269f68dcc0cd9653001630/server/routerlicious/packages/services-core/src/throttler.ts#L31

Reply:
Got it, but I think we should change it to retryAfterInSec or InMs.
How would the caller know how long to wait for? :/

Reply:
Then we have to change the INackContent under the protocol package, do we own this package? https://github.com/microsoft/FluidFramework/blob/6514933a426010f0ba269f68dcc0cd9653001630/server/routerlicious/packages/protocol-definitions/src/protocol.ts#L297

Reply:
I would propose renaming work, if we want to, be in a separate PR/issue since it pulls container-loader code into scope

Reply:
I agree. @muying-cui could you please create a new work item for this change? thanks :)

Reply:
I have created a task here: https://office.visualstudio.com/OE/_workitems/edit/4767053